tools/packaging/kernel/configs/fragments/common/cryptsetup.conf

```diff
@@ -0,0 +1,43 @@
+CONFIG_MD=y
+# CONFIG_BLK_DEV_MD is not set
+# CONFIG_BCACHE is not set
+CONFIG_BLK_DEV_DM_BUILTIN=y
+CONFIG_BLK_DEV_DM=y
+# CONFIG_DM_DEBUG is not set
```

Comment:
Is there a reason to keep those commented options?

Reply:
yes, this is auto generated, can be remove.